src/nupic/algorithms/backtracking_tm.py

```diff
@@ -1355,7 +1357,7 @@ def getSegmentOnCell(self, c, i, segIdx):
     return retlist
 
 
-  class _SegmentUpdate(Serializable):
+  class _SegmentUpdate(object):
```

Comment:
Perhaps a short comment to that effect to appease curious Georges in the future?